src/FileProviders/Manifest.MSBuildTask/test/Microsoft.Extensions.FileProviders.Embedded.Manifest.Task.Test.csproj

```diff
@@ -9,6 +9,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Reference Include="xunit" />
```

Comment:
Why was this necessary? Is it because this project doesn't end in "Tests"? If so, let's rename the test project.

Reply:
Yeah once a project is treated as a unit test project, arcade will nag you to remove this.